src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/CountFalsePositives.java

```diff
@@ -36,31 +39,28 @@
     @Argument(doc = "output file", fullName= "output", shortName = "O", optional = false)
     protected File output;
 
+
+    // TODO: eventually use tumor and normal sample names instead of the file name. To do so we must extract them from the vcf, which I don't know how.
     @Override
     public Object doWork() {
         Utils.regularReadableUserFile(vcfList);
-        String header = String.format("%s\t%s\t%s\n", "sample", "fps", "snp-or-indel");
-        StringBuilder sb = new StringBuilder(header);
-
-        try (
-                Stream<String> vcfFiles = Files.lines(Paths.get(vcfList.toString()));
-                PrintWriter outputWriter = new PrintWriter(new FileWriter(output))
-        ) {
-            sb = vcfFiles.reduce(sb,
-                    (acc, f) -> acc.append(countFalsePositivesAndBuildString(new File(f))),
-                    StringBuilder::append);
-            outputWriter.print(sb.toString());
+        try ( FalsePositiveTableWriter writer = new FalsePositiveTableWriter(output) ) {
+            List<String> vcfFilePaths = Files.readAllLines(Paths.get(vcfList.toString()));
+            List<File> vcfFiles = vcfFilePaths.stream().map(File::new).collect(Collectors.toList());
+            writer.writeAllRecords(vcfFiles);
```

Comment:
I think it's preferable to take all logic out of the `TableWriter`, so that you calculate false positives here and not inside `composeLine`.  This may require creating another nested class called eg `FalsePositiveRecord` that is basically a struct of the filename and snp and indel counts, but t's worth it.